pkgs/by-name/gx/gxml/package.nix

```diff
@@ -0,0 +1,58 @@
+{ fetchFromGitLab
+, glib
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, stdenv
+, lib
+, libgee
+, libxml2
+, vala
+}:
+stdenv.mkDerivation rec {
+  name = "gxml";
+  version = "0.20.3";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "gxml";
+    rev = version;
+    hash = "sha256-GlctGxsLyQ2kPV3oBmusRiouG4PPncBTh3vgxhVaQOo=";
+  };
+
+  outputs = [ "out" "dev" "devdoc" ];
```

Comment:
GNOME packages prefer having the `outputs` attribute between `version` and `src` see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887 and https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md.